deploy/platform/kubernetes/feature-rover/profiling.yaml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ebpf-profiling-trgigger
+spec:
+  schedule: "8 * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          containers:
+            - name: ebpf-profiling-trigger
+              image: ghcr.io/apache/skywalking-cli/skywalking-cli:d2c95c3782f137518b09554331794c10e1457d95
+              volumeMounts:
+              command:
+                - /bin/sh
+                - -c
+                - >
+                  /swctl --base-url=http://oap:12800/graphql profiling ebpf create fixed \
```

Comment:
The job is scheduled every 8 minutes but the duration is 9m. Would that be a problem?

Reply:
Changed to the scheduled every 5 minutes in an hour.
For now, we only support the On_CPU profiling, so I just trying to profile when the envoy has requested.